README.md

```diff
@@ -197,13 +197,13 @@ Coming from `react-use`? Check out our
 - #### Dom
 
   - [**`useClickOutside`**](https://react-hookz.github.io/web/?path=/docs/dom-useclickoutside--example)
-    — Triggers a callback when user clicks outside the target element.
+    — Triggers a callback when the user clicks outside a targeted element.
   - [**`useEventListener`**](https://react-hookz.github.io/web/?path=/docs/dom-useeventlistener--example)
-    — Subscribes an event listener to the target and automatically unsubscribes it on unmount.
+    — Subscribes an event listener to a target.
```

Comment:
I think we should be more consistent how we talk about target elements.

We could call them...
- "a target"
- "a target element"
- "a targeted element"
- "an element"
- "a chosen target"
- etc etc

What do you think?

Reply:
How about "a target element"? I think it would be good to include the word "element" there.

Reply:
I'm good with this!

Sorry for the delay, no idea how I didn't get pinged for your response